packages/system/src/theme.js

```diff
@@ -1,5 +1,5 @@
 import { css } from 'styled-components';
-import { resolve } from '@gympass/yoga-common';
+import { get } from 'lodash';
```

Comment:
I think this should be like this instead:

```suggestion
import get from 'lodash.get';
```

Using it from `lodash` means that we expect the package `lodash` as a dependency right? This only works because one of our dependencies has is as a dependency (dependency inception 😂 ).

Reply:
_blaming VSCode auto-import_